File: .sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { SwissFederalRailwaysSbbSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await SwissFederalRailwaysSbbSDK.test()
    equal(null !== testsdk, true)
  })

})

File: lua/test/exists_test.lua
-- SwissFederalRailwaysSbb SDK exists test

local sdk = require("swiss-federal-railways-sbb_sdk")

describe("SwissFederalRailwaysSbbSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
